pkg/pool-manager/pool_test.go

```diff
@@ -82,6 +82,28 @@ var _ = Describe("Pool", func() {
 			table.Entry("Start: 02:FF:00:00:00:00  End: 02:00:00:00:00:00", "02:FF:00:00:00:00", "00:00:00:00:00:00", true),
 		)
 
+		table.DescribeTable("should check that a mac address in within the range", func(startMacAddr, endMacAddr string, testMacAddr string, needToFail bool) {
+			startMacAddrHW, err := net.ParseMAC(startMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			endMacAddrHW, err := net.ParseMAC(endMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			testMacAddrHW, err := net.ParseMAC(testMacAddr)
+			Expect(err).ToNot(HaveOccurred())
+			isInRange := inRange(startMacAddrHW, endMacAddrHW, testMacAddrHW)
+			Expect(isInRange).To(Equal(needToFail))
+		},
+			table.Entry("Start: 02:00:00:00:00:00  End: 03:00:00:00:00:00 Mac-Tested: 02:00:00:01:00:00", "02:00:00:00:00:00", "03:00:00:00:00:00", "02:00:00:01:00:00", true),
```

Comment:
isn't the name "needToFail" , is a bit misleading here , since `Mac-Tested` is in range, it is actually should not fail?